doc/topics/installation/windows.rst

```diff
@@ -151,332 +157,233 @@ C. Update the Windows Service User for the ``salt-minion`` Service
-     are as follow:
+.. _create-build-environment:
 
-     * MarkupSafe
-     * Jinja
-     * MsgPack
-     * PSUtil
-     * PyYAML
-     * PyZMQ
-     * WMI
-     * Requests
-     * Certifi
+Create a Build Environment
+--------------------------
 
-     Open a command prompt and navigate to ``C:\Salt-Dev\temp``. Run the following
-     commands:
+1. Working Directory
+^^^^^^^^^^^^^^^^^^^^
 
-     .. code-block:: bat
+Create a ``Salt-Dev`` directory on the root of ``C:``. This will be our working
+directory. Navigate to ``Salt-Dev`` and clone the
+`Salt <https://github.com/saltstack/salt/>`_ repo from github.
 
-        pip install <cpuarch>\<MarkupSafe file name>
-        pip install <Jinja file name>
-        pip install <cpuarch>\<MsgPack file name>
-        pip install <cpuarch>\<psutil file name>
-        pip install <cpuarch>\<PyYAML file name>
-        pip install <cpuarch>\<pyzmq file name>
-        pip install <WMI file name>
-        pip install <requests file name>
-        pip install <certifi file name>
+Open a command line and type:
 
-2. Build the Salt Environment
+.. code-block:: bat
 
-   * Clone Salt
+    cd \
+    md Salt-Dev
+    cd Salt-Dev
+    git clone https://github.com/saltstack/salt
 
-     Open a command prompt and navigate to ``C:\Salt-Dev``. Run the following command
-     to clone salt:
+Go into the ``salt`` directory and checkout the version of salt to work with
+(2016.3 or higher).
 
-     .. code-block:: bat
+.. code-block:: bat
 
-        git clone https://github.com/saltstack/salt
+    cd salt
+    git checkout 2016.3
 
-   * Checkout Branch
+2. Setup the Python Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-     Checkout the branch or tag of salt you want to work on or build. Open a
-     command prompt and navigate to ``C:\Salt-Dev\salt``. Get a list of
-     available tags and branches by running the following commands:
+Navigate to the ``pkg\windows`` directory and execute the **build_env.ps1**
+PowerShell script.
 
-     .. code-block:: bat
+.. code-block:: bat
 
-        git fetch --all
+    cd pkg\windows
+    powershell -file build_env.ps1
 
-        To view a list of available branches:
-        git branch -a
+.. note::
+    You can also do this from Explorer by navigating to the ``pkg\windows``
+    directory, right clicking the **build_env.ps1** powershell script and
+    selecting **Run with PowerShell**
 
-        To view a list of availabel tags:
-        git tag -n
+This will download and install Python with all the dependencies needed to
+develop and build Salt.
 
-     Checkout the branch or tag by typing the following command:
+.. note::
+    If you get an error or the script fails to run you may need to change the
+    execution policy. Open a powershell window and type the following command:
 
-     .. code-block:: bat
+.. code-block:: powershell
 
-        git checkout <branch/tag name>
+    Set-ExecutionPolicy RemoteSigned
 
-   * Clean the Environment
+3. Salt in Editable Mode
```

Comment:
Maybe link to explain what an editable mode install is? https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs